.github/workflows/create-kustomize-with-bratiska-cli-inhouse.yml

```diff
@@ -0,0 +1,83 @@
+name: General workflow which creates kustomize using bratiska-cli.
+on:
+  workflow_call:
+    inputs:
+      runs-on:
+        description: 'Define the type of machine to run the job on'
+        type: string
+        required: false
+        default: '["self-hosted", "Linux", "X64", "bratislava"]'
+      directory:
+        description: 'A Folder within the repository where things should be build.'
+        default: '/'
+        required: false
+        type: string
+      debug:
+        description: 'Debug flag'
+        default: ''
+        required: false
+        type: string
+      version:
+        description: 'Bratiska-cli version'
+        default: 'stable'
+        required: false
+        type: string
+
+jobs:
+  create-kustomize-with-bratiska-cli:
+    name: Create kustomize with bratiska-cli
+    runs-on: ${{fromJSON(inputs.runs-on)}}
+    defaults:
+      run:
+        working-directory: ${{ inputs.directory }}
+    steps:
+      - name: Checking out
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Pipelines Version
+        run: |
+          echo "Pipelines version: 2.3.0"
+
+      - name: Directory check
+        run: pwd
+
+      - name: Print pipeline summary
+        run: |
+          echo "### Bratiska-cli is building:" >> $GITHUB_STEP_SUMMARY
+          echo ":arrow_right: Project folder: **${{ inputs.directory }}**" >> $GITHUB_STEP_SUMMARY
+
+      - name: Install Prerequisites -  Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '20'
+
+      - name: Installing Bratiska-cli version
+        run: yarn global add bratislava/bratiska-cli#${{ inputs.version }}
+
+      - name: Running git fetch for Bratiska-cli
+        run: git fetch
+
+      - name: Print Bratiska-cli version
+        run: bratiska-cli --version
+
+      - name: Creating kustomize file with Bratiska-cli
+        if: ${{ steps.secret-check.outputs.available != 'true' }}
+        run: bratiska-cli build_kustomize --env=staging ${{ inputs.flag }} ${{ inputs.debug }} --no_image_repo_check --image=image_url_placeholder
+        continue-on-error: false
```

Comment:
When customize file is built, can we save it as [artifact](https://github.com/actions/upload-artifact)? It can have lifetime of 2 days or something really small. But, it might help debug issues, during failing pipelines.